statics/js/components/workflow.js

```diff
@@ -153,19 +159,22 @@ Vue.component('workflow-call', {
       <div class="form-group">
         <workflow-params :workflow="currentWorkflow"></workflow-params>
       </div>
-      <div class="form-group" v-if="result.value">
+      <div class="form-group" v-if="display.result">
         <h1><span class="workflow-title">Result</span></h1>
-        <div class="form-group">
+        <div class="form-group" v-if="result.value">
           <textarea readonly id="workflow-output" type="text" class="form-control input-sm" rows="5" v-model="result.value" v-if="result.value && (typeof result.value) != 'object'"></textarea>
           <object-detail v-else-if="typeof result.value == 'object'" id="workflow-output" :object="result.value"></object-detail>\
         </div>
+        <div class="form-group" v-else>
+          <i class="fa fa-circle-o-notch fa-spin fa-2x fa-fw"></i>
+        </div>
       </div>
     </div>
   `,
 
   watch: {
     'currentWorkflow': function () {
-      this.result.value = undefined
+      this.display.result = false;
```

Comment:
Move it to the function: toggleReadyStatus

Reply:
done